tests/lib/rules/prefer-destructuring.js

```diff
@@ -34,6 +34,10 @@ ruleTester.run("prefer-destructuring", rule, {
       code: "var foo;"
     },
     {
+      code: "var foo = object.bar;",
+      parserOptions: { ecmaVersion: 6 }
+    },
```

Comment:
Can you add a test case for when the property type is a string literal (`object['bar']`)?